docs/source/getting_started.ipynb

```diff
@@ -155,6 +155,10 @@
     "\n",
     "2. Next, you may wish to edit or add some specific settings. The available configurations are:\n",
     "\n",
+    "- ``allow_list_rules``: A list of rules (whether custom of from Fixit) that should be applied to the repository. Omitting this setting allows all rules to run. For example:\n",
+    "\n",
+    "    allow_list_rules: [Flake8PseudoLintRule]\n",
+    "\n",
     "- ``block_list_patterns``: A list of patterns that indicate that a file should not be linted. For example::\n",
     "\n",
     "    block_list_patterns: ['@generated', '@nolint']\n",
```

Comment:
Can you also describe what will happen if a rule is added to both allow_list_rules and block_list_rules? That can help users understand the precedence order.